package.json

```diff
@@ -54,19 +56,28 @@
     "eslint-plugin-jsx-a11y": "6.5.1",
     "eslint-plugin-react": "7.29.4",
     "eslint-plugin-react-hooks": "4.5.0",
+    "expect": "1.20.2",
     "flow-bin": "0.133.0",
     "flow-copy-source": "2.0.9",
     "glamor": "2.20.40",
     "html-webpack-harddisk-plugin": "2.0.0",
     "html-webpack-plugin": "4.5.2",
-    "inferno": "7.4.11",
-    "nwb": "0.25.2",
+    "karma": "6.3.19",
+    "karma-chrome-launcher": "^3.1.1",
+    "karma-coverage": "2.2.0",
+    "karma-mocha": "2.0.1",
+    "karma-mocha-reporter": "2.2.5",
+    "karma-sourcemap-loader": "0.3.8",
+    "karma-webpack": "5.0.0",
+    "mocha": "10.0.0",
     "prettier": "2.6.2",
     "react": "16.14.0",
     "react-dom": "16.14.0",
     "regenerator-runtime": "0.13.9",
+    "rimraf": "3.0.2",
     "semantic-release": "17.4.7",
     "substyle-glamor": "4.1.2",
+    "vite": "2.9.7",
     "webpack": "4.46.0",
     "webpack-cli": "4.9.2"
```

Comment:
We probably should add `webpack-dev-server` to prevent this error:
![image](https://user-images.githubusercontent.com/1227637/167383350-927cbcf4-fb95-45cb-a351-023c78a08ece.png)

Reply:
Hmm, you're right, I should have went thought every command

Reply:
@tzimmermann fixed btw